guide/the-3d-window.md

```diff
@@ -5,8 +5,7 @@
 A single mouse click allows you to select a solid object.
 The bounding object of a selected solid is represented by white lines.
 These lines turn pink if the solid is colliding with another one and blue when the solid is idle, i.e., it comes to rest and it does not interact with any other active solid.
-Selecting a robot enables the `Show Robot Window` item in the `Tools` menu.
-Double-clicking on a solid object opens the Robot Window and if it was closed, the Scene Tree.
```

Comment:
> Double-clicking on a solid object opens [...] if it was closed, the Scene Tree.

Was this even working at some point? It doesn't seems to be the case neither on master or develop.

Reply:
Yes, I think that at some point (early Webots 7 or previous vesions) this was working.
In any case this is no longer the case now.

Reply:
Ok thank you, I will remove it for now and open an issue on Webots repository to discuss if we want to restore this feature or not.